src/Icons/README.md

```diff
@@ -0,0 +1,73 @@
+# Symfony UX Icons
+
+## Installation
+
+```bash
+composer require symfony/ux-icons
+```
+
+## Add Icons
+
+No icons are provided by this package. Add your svg icons to the `templates/icons/` directory.
```

Comment:
```suggestion
No icons are provided by this package. Add your svg icons to the `templates/icons/` directory and commit them.
```

I do think we need a mechanism in v1 to fetch icons, whether that's an entire pack or one-by-one.